session/pingpong/hermes_promise_storage.go

```diff
@@ -21,10 +21,13 @@ import (
 	"math/big"
 	"sync"
 
+	"github.com/asdine/storm/v3/codec/json"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/mysteriumnetwork/node/core/storage/boltdb"
 	"github.com/mysteriumnetwork/node/identity"
 	"github.com/mysteriumnetwork/payments/crypto"
 	"github.com/pkg/errors"
+	"go.etcd.io/bbolt"
```

Comment:
don't use storm?

Reply:
Why not, don't see a point to abstract DB lib as we don't use 2 abstractions. In tests we don't mock DB lib, but rather mock storm, by creating fixtures in it.